_test/test_sqw_file/test_file_input.m

```diff
@@ -178,30 +178,37 @@ function clearUp(obj)
             proj2.v=[1,1,0];
             
             s1_s=cut(obj.sqw2d_arr(2),proj2,[0.5,0.02,1],[0.9,1.1],[-0.1,0.1],[170,180]);
+            s1_f_h=cut(obj.sqw2d_name{2},proj2,[0.5,0.02,1],[0.9,1.1],[-0.1,0.1],[170,180]);
+            [ok,mess] = equal_to_tol(s1_s,s1_f_h,'ignore_str',1);
+            assertTrue(ok,['Memory based and file based cuts are different: ',mess])
+            
             s1_s_h=cut(obj.sqw2d_arr(2),proj2,[0.5,0.02,1],[0.9,1.1],[-0.1,0.1],[170,180]);
             s1_s_s=cut_sqw(obj.sqw2d_arr(2),proj2,[0.5,0.02,1],[0.9,1.1],[-0.1,0.1],[170,180]);
-            s1_f_h=cut(obj.sqw2d_name{2},proj2,[0.5,0.02,1],[0.9,1.1],[-0.1,0.1],[170,180]);
             s1_f_s=cut_sqw(obj.sqw2d_name{2},proj2,[0.5,0.02,1],[0.9,1.1],[-0.1,0.1],[170,180]);
             try
                 s1_s_d=cut_dnd(obj.sqw2d_arr(2),proj2,[0.5,0.02,1],[0.9,1.1],[-0.1,0.1],[170,180]);
                 failed=false;
             catch
                 failed=true;
             end
-            if ~failed, assertTrue(false,'Should have failed!'), end
+            assertTrue(failed,'Should have failed!');

```

Comment:
Its not this -- the code verifies any failure. It would be nice to check for specific failure, e.g. if `HORACE:cut_dnd:invalid_argument` had been thrown but I had a look inside and found in `cut_dnd` old Toby interface. 
I would not touch this at this stage.